README.md

```diff
@@ -1,2 +1,10 @@
 # instance-billing-type-check
 A CLI tool to determine if an instance is PAYG or BYOS across different CSPs
+
+## exit codes
+If the instance metadata could be verified successfully the exit code would be 10 (PAYG),
+however, if the instance metadata is present and no error ocurred during its verification
+and verification failed, exit code would be 11 (BYOS)
```

Comment:
From this it is hard why verification failure would not be an error. This should be re-worded to more clearly express the difference between a verification failure and an error during verification.
s/would be/is/